src/components/cars/Cars.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Carousel from 'react-multi-carousel';
+import 'react-multi-carousel/lib/styles.css';
+import { useSelector } from 'react-redux';
+import CarCard from './CarCard';
+import '../../assets/styles/Cars.css';
+import Navbar from '../navbar/Navbar';
+
+function Cars() {
```

Comment:
- [ ] After login, I'm immediately redirected to this page:

![image](https://user-images.githubusercontent.com/57421823/206781000-a77d7ce4-2353-484c-8bc5-34d2801c8281.png)

Therefore, I wasn't able to access the main content.

Tested on Chrome and Firefox.  Both in local dev and live deployments.  

If it is working for you, please check why is not working for new users.

If this is a bug, kindly fix it in order to ensure your app's correct functionality. 💪 

To help debugging, I'll share this from my localStorage: 

![image](https://user-images.githubusercontent.com/57421823/206782198-b99e028c-6dbf-40c9-8fee-306e9c7a2a99.png)

my user password is 03030303